SourceX/DiabloUI/errorart.h

```diff
@@ -0,0 +1,339 @@
+#pragma once
```

Comment:
Consider splitting moving the data int a `cpp` file to improve compilation speed (better "caching", don't need to reparse/recompile this file)

Reply:
Oh I actually thought it would be faster if it was in a header (as long as it isn't changed), thanks.